packages/lit-protocol-cipher/test/index.test.ts

```
@@ -147,34 +158,145 @@ describe('LitProvider', () => {
   });
 
   describe('getSessionSignatures', () => {
-    it('should get session signatures successfully', async () => {
+    beforeEach(() => {
+      (LitAccessControlConditionResource as unknown as jest.Mock).mockClear();
+      (createSiweMessage as jest.Mock).mockClear();
+
+      // Mock getSessionSigs to call the authNeededCallback
+      (mockLitClient.getSessionSigs as jest.Mock).mockImplementation(async (args: unknown) => {
+        const { authNeededCallback } = args as { authNeededCallback: Function };
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)